ui/src/main/webapp/package.json

```diff
@@ -60,23 +68,13 @@
     "showdown": "^1.5.1",
     "text-loader": "^0.0.1",
     "webpack-merge": "^0.15.0",
-    "zone.js": "^0.7.6",
-    "d3": "4.4.0",
-    "d3-transition": "1.0.3",
-    "d3-selection-multi": "1.0.0",
-    "ngx-charts": "^3.0.2",
-    "angular2-moment": "^1.1.0"
+    "zone.js": "^0.7.6"
```

Comment:
Hmm I'm thinking, maye it is better idea to create that separate PR with just changing the order right away. (And keep it in this PR as it is).
It will simplify changes for both your and Ondra's PRs and it will also reduce possibility of rebase errors.